tests/test_base.py

```diff
@@ -0,0 +1,29 @@
+import pkgutil
+import pytest
+
+import stdlib_list
+
+
+@pytest.mark.parametrize(
+    ("version", "canonicalized"),
+    [("2.7", "2.7"), ("3.10", "3.10")],
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏add text cases that don't map 1-to-1 - such as 3.4.3 -> 3.4 to exercise the `[:2]` .

Reply:
Good call, fixing!

Reply:
185e894..92e5670